app/src/androidTest/java/de/westnordost/streetcomplete/util/NameAndLocationLabelTest.kt

```diff
@@ -45,8 +45,6 @@ class NameAndLocationLabelTest {
         )))
     }
 
-    // https://github.com/streetcomplete/StreetComplete/issues/2640
-    @Ignore("https://github.com/streetcomplete/StreetComplete/issues/4916")
```

Comment:
I think the first comment should stay, to give some context what exactly is tested here (similar to the other test cases). But the `@Ignore(…)` can of course be deleted.